CONTRIBUTING.md

```diff
@@ -163,7 +163,77 @@ in general try to follow them.
+OpenTelemetry community members, and anyone else who would like to contribute.
+
+#### Triagers
+
+Contributors with [triager](https://github.com/open-telemetry/opentelemetry-collector-contrib/#contributing) permissions can help move issues along is through adding missing component labels,
+which helps organize issues and triggers automations to notify code owners. They can also use their familiarity with the Collector and
+its components to investigate issues themselves. Alternatively, they may point issue authors to another resource or someone else who may know more.
+
+#### Code Owners
+
+In many cases, the code owners for an issue are the best resource to help determine the root cause of a bug or whether an enhancement
+is fit to be added to a component. Code owners will be notified by repository automations when:
+
+* a component label is added to an issue
+* an issue is opened
+* the issue becomes stale
+
+Code owners may not have triager permissions on the repository,
+so they can help triage through investigation and by participating in discussions. They can also help organize issues by
+[adding labels via comments](#adding-labels-via-comments).
+
+#### Community Members
+
+Community members or interested parties are welcome to help triage issues by investigating the root cause of bugs, adding input for
+features they would like to see, or participating in design discussions.
+
+### Triage process
+
+Triaging an issue requires getting the issue into a state where there is enough information available on the issue or understanding
+between the involved parties to allow work to begin or for the issue to be closed. Facilitating this may involve, but is not limited to:
+
+- Determining whether the issue is related to the code or documentation, or whether the issue can be resolved without any changes.
+- Ensuring that a bug can be reproduced, and if possible, the behavior can be traced back to the offending code or documentation.
+- Determining whether a feature request belongs in a component, should be accomplished through other means, or isn't appropriate for a component at this time.
+- Guiding any interested parties to another person or resource that may be more knowledgeable about an issue.
+- Suggesting an issue for discussion at a SIG meeting if a synchronous discussion would be more productive.
+
+#### Issue assignment
+
+Issues are assigned for someone to work on by a triager when someone volunteers to work on an issue. Assignment is intended to prevent duplicate work by making it visible who is
+working on a particular task. A person who is assigned to the issue may be assigned to help triage the issue and implement it, or can be assigned after the issue has already been
+triaged and is ready for work. If someone who is assigned to an issue is no longer able to work on it, they may request to be unassigned from the issue.
+
+### Label Definitions
```

Comment:
(not necessarily on this PR) a couple other labels we may want to define:
-  `release:blocker`
- `Vendor Specific Component`

Reply:
These are both good to have, I've added them. Let me know if you'd like any changes to be made to the explanations.